Source/WebCore/page/SettingsBase.cpp

```diff
@@ -496,4 +496,12 @@ void SettingsBase::resourceUsageOverlayVisibleChanged()
 #endif
 }
 
+#if HAVE(AVCONTENTKEYSPECIFIER)
+void SettingsBase::sampleBufferContentKeySessionSupportEnabledChanged()
+{
+    if (m_page)
+        MediaSessionManagerCocoa::setSampleBufferContentKeySessionSupportEnabled(m_page->settings().sampleBufferContentKeySessionSupportEnabled());
```

Comment:
I guess EWS will tell, but does this need an include for `MediaSessionManagerCocoa.h` ?

Reply:
Oh, duh. Yes.